src/app/routes/topic/getInitialData/index.js

```diff
@@ -1,11 +1,37 @@
+import { BFF_FETCH_ERROR } from '#lib/logger.const';
+import { INTERNAL_SERVER_ERROR } from '#lib/statusCodes.const';
+import nodeLogger from '#lib/logger.node';
 import { fixturePromos } from '#pages/TopicPage/fixtures';
 
-export default () => {
-  return {
-    status: 200,
-    pageData: {
-      title: 'hello world',
-      promos: fixturePromos(),
-    },
-  };
+const logger = nodeLogger(__filename);
+
+const validateResponse = async response => {
+  const { status } = response;
+  if (response.status === 200) {
+    const { data } = await response.json();
+    return { data, status };
+  }
```

Comment:
How are invalid topic responses handled e.g. for an invalid service - variant pair? I'd expect us to propogate a 404 from the BFF to return a 404 page to the user. 

Can we add a test for this?